src/store/edit_post_form.js

```diff
@@ -29,7 +29,7 @@ const initialState = i.fromJS({
 export default function reducer(state = initialState, action) {
   switch (action.type) {
     case posts.RESET_EDIT_POST_FORM: {
-      state.set(action.post_id, i.fromJS({
+      state.set(action.payload.post_id, i.fromJS({
```

Comment:
Yep. This clearly shouldn't be this way. It didn't even make sense before the changes.